src/main/kotlin/no/nav/syfo/api/v2/domain/oppfolgingsplan/OpprettOppfolgingsplan.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.syfo.api.v2.domain.oppfolgingsplan
+
+data class OpprettOppfolgingsplanRequest (
+    val sykmeldtFnr: String? = null,
```

Comment:
Hvorfor er sykmeldtFnr nullable? Dette bør vel være påkrevd for å opprette en plan

Reply:
Det er fordi sykmeldtFnr bare brukes dersom det er AG som oppretter plan. I SM sitt tilfelle så er det innloggetFnr som benyttes. Vi kan evt lage det påkrevet og bare bruke det for AG.

Reply:
Det kan kanskje vært lurt :)

Reply:
Har fikset nå